SimpleChat/Application/Scenes/Home/ViewControllers/HomeViewController.swift

```diff
@@ -18,10 +18,28 @@ final class HomeViewController: UIViewController, StoryboardInstanceable {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        
         setUpNavigationBar()
         setUpCollectionView()
         bind()
-        viewModel.requestContactsPermission()
+        
+        if let contactsPermissionGranted = viewModel.isContactsPermissionGranted.value {
```

Comment:
We could change this to be `viewModel.isContactsPermissionGranted.value!` to force behavior from the view model. For now, this is fine.